crates/context/config/src/client/macros.rs

```diff
@@ -1,12 +1,11 @@
-#[macro_export]
 macro_rules! transport {
     // Type declaration
     (
         $(#[$meta:meta])*
-        pub type $name:ident = ( $($t:ty),* $(,)? );
+        $($pub:vis)? type $name:ident = ( $($t:ty),* $(,)? );
```

Comment:
okay some things are unexpectedly broken [with visibility matching](https://fprijate.github.io/tlborm/pat-visibility.html#matching-and-substituting), you can revert to what you had before, we'll revisit this along with lifetimes in a later iteration